kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/Packages/PackagesPanelUi.java

```diff
@@ -122,6 +127,24 @@
     @UiField
     Well marketplaceInstallWell;
 
+    @UiField
```

Comment:
Can these UIField be private?
Maybe also marketplaceDescriptor can be private aswell